dll/win32/kernel32/kernel32.spec

```diff
@@ -433,6 +433,7 @@
 @ stdcall -norelay GetCurrentProcess()
 @ stdcall -norelay GetCurrentProcessId()
 @ stdcall GetCurrentProcessorNumber() ntdll.RtlGetCurrentProcessorNumber
+@ stdcall GetCurrentPackageId(ptr ptr)
```

Comment:
Please add the `-version=0x602+` option

Reply:
I understand, but if we don't export this API from kernel32 now, then this PR will have no effect towards the purpose I opened it (which is getting a class of installer to work on ReactOS, today). IMHO we should add the version flag only after #1872 is merged. If we want to put this PR on hold until then, that's fine.

Reply:
Applications most likely wont call this function if it's not present.

Reply:
But the .NET Core installer ([link to download](https://dotnet.microsoft.com/download/thank-you/dotnet-runtime-3.0.0-windows-x86-installer)) does just that. I wouldn't have made this PR unless this was an immediate problem for me. Run the file linked above on ReactOS without this patch, click the Quit button after the error message appears, and observe the crash.

Reply:
You are not reading what I say, or not understanding what I say:
if the function is not present (NO EXPORT PRESENT) it most likely will not be called.
So if the function is present (but a stub) it will be called, which causes an exception.